library/alloc/src/vec/extract_if.rs

```diff
@@ -24,12 +24,13 @@ pub struct ExtractIf<
     T,
     F,
     #[unstable(feature = "allocator_api", issue = "32838")] A: Allocator = Global,
-> where
-    F: FnMut(&mut T) -> bool,
+>
 {
     pub(super) vec: &'a mut Vec<T, A>,
```

Comment:
Do we still need all these fields to be `pub(super)` now that there's a local `fn new`?

Reply:
removed the visibility.